### PR TITLE
Fix return key not working #244

### PR DESCRIPTION
**Pull request by Waqar144**
#244

### CODE REVIEW

File: src/mainwindow.cpp [hunk 1]
```diff
@@ -2753,7 +2753,7 @@ bool MainWindow::eventFilter(QObject *object, QEvent *event)
         auto *keyEvent = static_cast<QKeyEvent *>(event);
         if (keyEvent->key() == Qt::Key_Return && m_searchEdit->text().isEmpty()) {
             setFocusOnText();
-            return true;
+			break;
```
Comment (bjorn) [inline src/mainwindow.cpp]: Just suggesting to keep the indentation consistent:
```suggestion
            break;
```

Comment (Waqar144) [reply src/mainwindow.cpp]: Alright, thanks.

Comment (Waqar144) [reply src/mainwindow.cpp]: How about we just not put a break in here? There's a `break` after the 'if` condition which will do the same.

**Comment by Waqar144** [comment]
Also fixed row color not changing when clicking on `textEdit`. I must admit that the `if` ladder here is rather perplexing:

https://github.com/nuttyartist/notes/blob/c7a8687abd2c73996b05a17f35f0d43b11805ee5/src/mainwindow.cpp#L2670-L2674

**Comment by nuttyartist** [comment]
> @nuttyartist
> @bjorn
> Also fixed row color not changing when clicking on `textEdit`. I must admit that the `if` ladder here is rather perplexing:
> 
> https://github.com/nuttyartist/notes/blob/c7a8687abd2c73996b05a17f35f0d43b11805ee5/src/mainwindow.cpp#L2670-L2674

Hahaha

Cool, merging!

**Pull request merged**